### ISSUE TITLE
Privacy concerns regarding the Portuguese sentence collection

### ISSUE THREAD

**Issue by limapedro** [mozilla/common-voice#2970]
Hi, I'd like to issue that while I was recording some clips yesterday I saw the legal names were in the sentence collection, I since then stopped recording until I got confirmation that everthing is fine, does Mozilla have a policy on using legal names of real people in the dataset? I don't think generic names would be a problem, but full names of people can be problematic.
Please reply me as soon as you can so we can sort this out.

Happy Christmas and New Year everyone!!!

**Comment by phirework** [comment]
Hello, can you provide more information about what those names were? Many languages use sentences that are drawn from Wikipedia, and so if an article has the full name of a person, it may be included here as well. If you are concerned about specific privacy violations, you can also send us an email at commonvoice@mozilla.org, or report that sentence.

**Comment by limapedro** [comment]
@phirework I don't think we have sentences extracted from Wikipedia yet for Portuguese, someone added this through the sentence collection tool. Just as to be sure, would people be ok with their full name on the internet in other countries? I was recording some sentences and I didn't felt comfortable reading people's name. The names are in this file: https://github.com/mozilla/common-voice/blob/main/server/data/pt/sentence-collector.txt to search for full names you can type some common Brazilians last names such 'silva', 'Oliveira' or 'souza' to see what I'm talking about.

**Comment by ftyers** [comment]
I think it depends on how common the name is, for example I would probably be uncomfortable with Francis Morton Tyers, but probably not if my name was Jane Doe or John Smith or Albert Einstein or Marie Curie.  It could also be that some of the names are made up, e.g. I tried searching for a "Aldo da Silva Maciel" but found no results. 

I think it is important to have proper names (including both person names and place names) in the corpus, as they can often have unpredictable pronunciations. But they should probably be restricted to frequent ones, or made up ones.

**Comment by phirework** [comment]
Hmm, it looks like a bunch of the Sentence Collector sentences for Portuguese have "Lista criada de nomes" as a source. It sounds like these may have been autogenerated - @MichaelKohler, do you have any more insight as to who submitted this, and from where?

**Comment by MichaelKohler** [comment]
> Hmm, it looks like a bunch of the Sentence Collector sentences for Portuguese have "Lista criada de nomes" as a source. It sounds like these may have been autogenerated - @MichaelKohler, do you have any more insight as to who submitted this, and from where?

As I don't have direct access to the database, I can't say who uploaded these.

**Comment by phirework** [comment]
I will check with Alberto when he's back, thanks!

**Comment by marifeitosa** [comment]
Hello I am new here, I would like to contribute something that is not code. 
It can be something related to documentation, or translation of project documentation to Portuguese... I look forward to hearing back.

**Comment by marifeitosa** [comment]
@limapedro Hi, I'm sure I'll look at what paths I should follow to make these contributions. Health!
Thank you!

**Comment by marifeitosa** [comment]
@limapedro Hi, How to validate?
Can you provide the link for validation?

Thank you!

**Comment by ftyers** [comment]
Close as stale.
